SkyBlock/README.md

```diff
@@ -0,0 +1,358 @@
+# SkyBlock
+
+![](Skyblock_Island.webp "SkyBlock Island")
+
+## Introduction
+
+**Skyblock** is one of the most popular survival gamemodes. The idea is that you can create and expand your very own world infinitely with only minimal materials. You start on a small island floating in the middle of void with a few items and a single tree and you must learn to use and preserve your materials wisely as each one has a strategic use, (for example lava buckets), and to know the basics of Minecraft in order to complete challenges and to expand. It is normal to make beginner mistakes, sometimes requiring many restarts.
+
+## Commands
+
+On TrixMC we have a set of commands that you can use to manage your SkyBlock world. The commands are of the format `/is <command>`. The commands are listed below:
+
+### `controlpanel`
+
+`/is controlpanel` is an all-in-one command that opens a GUI that allows you to manage your TrixMC SkyBlock world.
+
+### `ban`
+
+`/is ban <player>` is used to ban a player from your Island.
+
+### `banlist`
+
+`/is banlist` is used to list all players banned from your island.
+
+### `biomes`
+
+`/is biomes` is used to buy and set the biome for your island. More info about biomes can be found in [Biomes](#biomes-1).
+
+### `challenges`
+
+`/is challenges` is used to view the challenges. More info about challenges can be found in [Challenges](#challenges-1).
+
+### `create`
+
+`/is create` is used to create your island.
+
+### `fly`
+
+`/is fly` is used to toggle flying on your island. More info about fly can be found in [Fly](#fly-1).
+
+### `generator`
+
+`/is generator` is used to buy and upgrade your cobblestone generator. More info about generator can be found in [Generator](#generator-1).
+
+### `go`
+
+`/is go` is used to teleport to your island.
+
+### `info`
+
+`/is info` is used to view island information.
+
+### `language`
+
+`/is language` is used to change the language.
+
+### `level`
+
+`/is level <player>` is used to calculate your island level or mentioned player's level.
+
+### `reset`
+
+`/is reset` is used to reset your island. WARNING: ALL PROGRESS WILL BE LOST IF YOU USE THIS COMMAND.
+
+### `sethome`
+
+`/is sethome` is used to set your home teleport point.
+
+### `setname`
+
+`/is setname` is used to set the name of your island.
+
+### `settings`
+
+`/is settings` is used to view and change your island settings. More info about settings can be found in [Settings](#settings-1).
+
+### `resetname`
+
+`/is resetname` is used to reset the name of your island.
+
+### `unban`
+
+`/is unban <player>` is used to unban a player from your Island.
+
+### `team`
+
+1. `/is team` is used to manage your team.
```

Comment:
I think a bulleted list would make more sense here.

Reply:
This way commands are on the sidebar and it helps in easy navigation.